src/test/java/com/spotify/docker/client/CompressedDirectoryTest.java

```diff
@@ -71,8 +71,8 @@ public void testFileWithIgnore() throws Exception {
         final String name = entry.getName();
         names.add(name);
       }
-      assertThat(names, containsInAnyOrder("Dockerfile", "bin/date.sh", "subdir2/keep-me",
-                                           ".dockerignore"));
+      assertThat(names, containsInAnyOrder("Dockerfile", "bin/date.sh", "subdir2/keep.me",
```

Comment:
We should also check the size of `names` to ensure we're not including files that should be ignored.